From 7389814a22f66630ba5424020f7ddbb062f3969b Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 22:57:52 +0800
Subject: [PATCH 50/58] Revert "drm/bridge: nwl-dsi: Fix PM disable depth
 imbalance in nwl_dsi_probe"

This reverts commit d9d61beb21d703e0d2d93314ca0b70d63c210378.
---
 drivers/gpu/drm/bridge/nwl-dsi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/nwl-dsi.c b/drivers/gpu/drm/bridge/nwl-dsi.c
index b68d335981588..6cac2e58cd15f 100644
--- a/drivers/gpu/drm/bridge/nwl-dsi.c
+++ b/drivers/gpu/drm/bridge/nwl-dsi.c
@@ -1188,7 +1188,6 @@ static int nwl_dsi_probe(struct platform_device *pdev)
 
 	ret = nwl_dsi_select_input(dsi);
 	if (ret < 0) {
-		pm_runtime_disable(dev);
 		mipi_dsi_host_unregister(&dsi->dsi_host);
 		return ret;
 	}
-- 
2.27.0

